lib/apiservers/engine/backends/container/viccontainer.go

```diff
@@ -30,32 +29,24 @@ type VicContainer struct {
 	HostConfig  *containertypes.HostConfig
 
 	m     sync.RWMutex
-	execs map[string]*types.ExecConfig
+	execs map[string]struct{}
```

Comment:
:) We don't keep ExecConfig because we don't need them. When we create we persist that data on the portlayer side and later when we need them we simply call inspect